### PR TITLE
[gui] follow up for PR #11704

### PR DESCRIPTION
**Pull request by xhaggi**
@ksooo as discussed at #11704.

### CODE REVIEW

**Comment by ksooo** [comment]
You should change "actualitems" to "numactualitems" as well, then, for consistency reasons. Otherwise I see no point in the name change of "nonfolderitems" to "numnonfolderitems".

**Comment by xhaggi** [comment]
Added the missing break, thanks.

> You should change "actualitems" to "numactualitems" as well, then, for consistency reasons. Otherwise I see no point in the name change of "nonfolderitems" to "numnonfolderitems".

That's the reason why I want to double check new GUI labels etc. If it was merged it becomes hard to change. @phil65 does the same mistake if you ask me. Nothing will point-out that this info label will return the size of items. That's the reason why `NumItem` is prefixed with `num`. We should use the same pattern for new info labels if it makes sense and this is the case here.

**Comment by xhaggi** [comment]
I've added a separate commit which renames the info label `ActualItems` to `NumAllItems`. It leaves the old one in for backward compatibility.

**Comment by xhaggi** [comment]
I've dropped the backward compat stuff because `ActualItems` was merged into Leia only. In that case we don't need it.

**Comment by xhaggi** [comment]
Jenkins build this please

**Comment by ksooo** [comment]
Have you seen my latest comments? I'm not sure the PR is ready to merge.

**Comment by xhaggi** [comment]
Which one?

**Comment by ksooo** [comment]
Seems I forgot to submit my review. ;-)

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -2407,7 +2407,7 @@ const infomap container_bools[] ={{ "onnext",           CONTAINER_MOVE_NEXT },
                                   { "numpages",         CONTAINER_NUM_PAGES },
                                   { "numitems",         CONTAINER_NUM_ITEMS },
                                   { "numnonfolderitems", CONTAINER_NUM_NONFOLDER_ITEMS },
-                                  { "actualitems",      CONTAINER_ACTUAL_ITEMS },
+                                  { "numallitems",      CONTAINER_NUM_ALL_ITEMS },
```
Comment (xhaggi) [inline xbmc/GUIInfoManager.cpp]: Okay, lets ping our skinners @ronie @phil65 any objections?

File: xbmc/guilib/GUIBaseContainer.cpp [hunk 2]
```diff
@@ -1304,27 +1304,27 @@ std::string CGUIBaseContainer::GetLabel(int info) const
...
+      for (auto item : m_items)
+      {
+        if (!item->m_bIsFolder)
+          numItems++;
+      }
+      label = StringUtils::Format("%u", numItems);
+    }
+    break;
   default:
       break;
```
Comment (xhaggi) [inline xbmc/guilib/GUIBaseContainer.cpp]: Sure, but in the same commit. I don't want to add a new one only for that.

File: xbmc/GUIInfoManager.cpp (hunk 1 above)
Comment (xhaggi) [inline xbmc/GUIInfoManager.cpp]: Btw I don't have a strong opinion on that, but all means all :smile:

File: xbmc/guilib/GUIBaseContainer.cpp (hunk 2 above)
Comment (ksooo) [inline xbmc/guilib/GUIBaseContainer.cpp]: sure

File: xbmc/GUIInfoManager.cpp (hunk 1 above)
Comment (ronie) [inline xbmc/GUIInfoManager.cpp]: no objections from my end

**Comment by xhaggi** [comment]
jenkins build this please

**Pull request merged**